[2021-09-27 15:50:02,601] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T15:40:00+00:00 [queued]>
[2021-09-27 15:50:02,638] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T15:40:00+00:00 [queued]>
[2021-09-27 15:50:02,640] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:50:02,641] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 15:50:02,643] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:50:02,680] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): say_hello> on 2021-09-27T15:40:00+00:00
[2021-09-27 15:50:02,690] {standard_task_runner.py:52} INFO - Started process 35098 to run task
[2021-09-27 15:50:02,701] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'say_hello', '2021-09-27T15:40:00+00:00', '--job-id', '98', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp3q8h7w76', '--error-file', '/tmp/tmpl968v1ve']
[2021-09-27 15:50:02,713] {standard_task_runner.py:77} INFO - Job 98: Subtask say_hello
[2021-09-27 15:50:02,859] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.say_hello 2021-09-27T15:40:00+00:00 [running]> on host 196580edc338
[2021-09-27 15:50:03,013] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Attasil Chitwirote
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=say_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T15:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-27T15:40:00+00:00
[2021-09-27 15:50:03,020] {my_dag.py:9} INFO - Hello SAKSIAM.
[2021-09-27 15:50:03,023] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 15:50:03,069] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=say_hello, execution_date=20210927T154000, start_date=20210927T155002, end_date=20210927T155003
[2021-09-27 15:50:03,193] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 15:50:03,405] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
